kube-config-files/kafka-service.yaml

```diff
@@ -0,0 +1,110 @@
+###############################################################################
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kafka
+  labels:
+    app: kafka
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka
+  template:
+    metadata:
+      labels:
+        app: kafka
+        visualize: "true"
+    spec:
+      containers:
+      - name: kafka
+        image: wurstmeister/kafka:0.8.2.0
+        env:
+        - name: KAFKA_MESSAGE_MAX_BYTES
+          value: "16777216"
+        - name: KAFKA_REPLICA_FETCH_MAX_BYTES
+          value: "16777216"
+        - name: DOCKER_HOST
+          value: "localhost:2375"
+        - name: KAFKA_ADVERTISED_HOST_NAME
+          value: "kafka"
+        - name: KAFKA_ZOOKEEPER_CONNECT
+          value: "localhost:2181"
+        - name: KAFKA_BROKER_ID
+          value: "1"
+        - name: KAFKA_PORT
+          value: "9092"
+        - name: KAFKA_ADVERTISED_PORT
+          value: "9092"
+        ports:
+        - containerPort: 9092
+        volumeMounts:
+         - name: kafka-persistent-storage
+           mountPath: /kafka
+           subPath: kafka-data
+      - name: zookeeper
+        image: wurstmeister/zookeeper
+        ports:
+        - containerPort: 2181
+        volumeMounts:
+        - name: kafka-persistent-storage
+          mountPath: /opt/zookeeper-3.4.6/data
+          subPath: zk-data
+      - name: kafka-proxy
+        image: coco/kafka-proxy:v1.0.0
+        imagePullPolicy: Always
+        env:
+        - name: RP_FETCH_MESSAGE_MAX_BYTES
+          value: "16777216"
+        - name: RP_CONSUMER_REQUEST_MAX_BYTES
+          value: "17825792"
+        - name: RP_ZOOKEEPER_CONNECT
+          value: "localhost:2181"
+        - name: RP_MAX_REQUEST_SIZE
+          value: "17825792"
+        ports:
+        - containerPort: 8082
+      volumes:
+       - name: kafka-persistent-storage
+         awsElasticBlockStore:
+           volumeID: vol-0a7ee795314cde11c
```

Comment:
Please update the provisioning process described here https://docs.google.com/document/d/1TTih1gcj-Vsqjp1aCAzsP4lpt6ivR8jDIXaZtBxNaUU/edit?pli=1#heading=h.idonu4gksr10, and specify the provisioning of the EBS volume , as we need to keep track of what is needed for a full provisioning of an environment